### PR TITLE
Atmospheric readings are now accurate to 0.01%

### LINKED ISSUE #16329: PDA Atmospheric scanner is casting all percentages to int
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))
Plasma is dangerous at values lower than 0.5% however the Atmospheric scanner function on pdas doesn't display decimals, making it impossible to ensure that a contaminated area is truly plasma free.
[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

### PR DESCRIPTION
**Pull request by Tytynguyen**
*Atmospheric readings are rounded to the 0.01%
+Atmospheric analyzer will show an alert if plasma concentration gets above 0.5%

Fixes #16329

### CODE REVIEW

**Comment by KazeEspada** [comment]
How about 0.01 cause thats air alarms I think.

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -281,28 +281,30 @@ MASS SPECTROMETER
 		environment.garbage_collect()
 
 		if(abs(n2_concentration - N2STANDARD) < 20)
-			user << "<span class='info'>Nitrogen: [round(n2_concentration*100)] %</span>"
+			user << "<span class='info'>Nitrogen: [round(n2_concentration*100, 0.01)] %</span>"
```
Comment (Cheridan) [inline code/game/objects/items/devices/scanners.dm]: did you mean these to be 0.01 or 0.001? 

would be nice if you made these into a define, either way.

Comment (duncathan) [inline code/game/objects/items/devices/scanners.dm]: yeah, a define would be best.

Comment (duncathan) [inline code/game/objects/items/devices/scanners.dm]: I'll do this myself later I'll be touching these soon anyway

**Pull request merged**